### PR TITLE
Fixes content width override

### LINKED ISSUE #3: Change width of content
Set width of default size to 700px / 70rem.

### PR DESCRIPTION
**Pull request by ChaosExAnima**
Fixes #3.

Screen Shot 2020-05-07 at 6 25 12 PM

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -87,6 +87,29 @@ a {
 	font-family: var( --font-body );
 }
 
+.color-accent,
```
Comment (jffng) [inline style.css:90]: Are these accent color styles meant to be bundled in this PR?

File: style.css [hunk 2]
```diff
@@ -211,6 +234,115 @@ a {
...
+	}
+
+	.entry-content > p .alignright,
+	.entry-content > .wp-block-image .alignright,
+	figure.wp-block-gallery.alignright {
+		margin-right: calc(( 100vw - var( --default-content-width ) - 8rem ) / -2);
+	}
+}
+
+@media ( min-width: 1000px ) {
```
Comment (jffng) [inline style.css:269]: Is there content or a specific page that you'd recommend testing these alignment styles against?

File: style.css (hunk 1 above)
Comment (ChaosExAnima) [reply style.css:90]: Yeah, I noticed them missing when I was looking at author pages. Wasn't worth starting a new branch imo.

File: style.css (hunk 2 above)
Comment (ChaosExAnima) [reply style.css:269]: Images are the best way to test here.

**Comment by ChaosExAnima** [comment]
@jffng - I tested adding block editor styles, but increasing the width to 700px broke a few things. Interestingly the 580px width comes from Gutenberg core, not the parent theme.

**Pull request merged**